fixtures/README.md

```diff
@@ -1,7 +1,5 @@
 # Fixtures
 
-This folder contains static test fixtures that are hosted by this test suite.
+This folder contains static BitstringStatusList test fixtures referenced by external test-suites.
 
-## revocationList.json
-
-Is a Verifiable Credential based revocation list conforming to the [RevocationStatusList2020](https://w3c-ccg.github.io/vc-status-rl-2020/) spec, whereby the credential associated to index 1 is revoked and all other credentials described by the other indexes represented in the list are not revoked.
\ No newline at end of file
+All status credentials have the following indexes set to 1 `201` `94023` `57444` and should be used for their respective purposes.
```

Comment:
```suggestion
All status credentials should be used for their respective purposes, and have the following indexes set to 1: `201`, `94023`, `57444`.
```

Reply:
Removing this directory, I will apply your suggestion in the bitstring status list test suite repo. Thank you